nlp.py

```diff
@@ -52,7 +52,7 @@ def isa(self, word, cat):
         return cat in self.categories[word]
 
     def __repr__(self):
-        return '<Grammar %s>' % self.name
+        return '<Grammar {0!r}>'.format(self.name)
```

Comment:
Can you please elaborate on why are using {0!r} instead of {} given self.name is not a special object for which we have defined __repr__